<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindowJoin</class>
 <widget class="QDialog" name="SettingsWindowJoin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsWindowJoin</string>
  </property>
  <widget class="QPlainTextEdit" name="WarriorEditor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fileButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>From File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verifyButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="joinButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Join Session</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIP">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
